src/archetypes/referencebrowserwidget/browser/popup.pt

```diff
@@ -192,9 +192,12 @@
         </td>
 
         <td tal:condition="image_portal_types">
-          <img tal:condition="python: item.Type in image_portal_types"
-               tal:attributes="src string:${item/getURL}/$image_method"
-               />
+          <label tal:condition="referenceable"
```

Comment:
having a condition on the label means: do not show an image for referencable types. shouldn't  we use 

```
<label tal:omit-tag="not: referenceable" ...>
```

to make the image show up even if the item can't be referenced?